script/semantic-conventions/semconv.sh

```diff
@@ -9,20 +9,18 @@
 # Source repositories:
 #  - https://github.com/open-telemetry/semantic-conventions/releases
 #  - https://github.com/open-telemetry/build-tools/releases
-set -e
+set -ex
```

Comment:
Do we want to write a trace for each command before execution?

Reply:
It doesn't hurt, and is quite helpful for debugging. Other SIGs which I took inspiration from also have this set...